lib/elixir/pages/syntax-reference.md

```diff
@@ -374,6 +374,28 @@ end
 #=> {:fn, [], [{:->, [], [[1, 2], 3]}, {:->, [], [[4, 5], 6]}]}
 ```
 
+### Access syntax
+
+The access syntax (`foo[:bar]`) is represented by a `{:., [], [Access, :get]}` call, and the arguments are the ones used to call `Access.get/3`:
+
+```elixir
+quote do
+  foo[:bar]
+end
+#=> {{:., [], [Access, :get]}, [], [{:foo, [], Elixir}, :bar]}
+```
+
```

Comment:
This is shown later in the document as syntax sugar (as it translates to existing code):

```suggestion
```